data_visualization.py

```diff
@@ -0,0 +1,252 @@
+#!/opt/anaconda3/bin/python
+
+"""
+This script is used to visualize EEG and MEG signals.
```

Comment:
Mention here that the final visualization will be done through the MNE framework and add the link to their website